.travis.yml

```diff
@@ -55,14 +64,17 @@ install:
     - echo $PATH
     - bids-validator --version
     - which bids-validator
-    - conda env create -f environment.yml --name=testenv
-    - source activate testenv
-    - pip uninstall --yes --quiet mne  # uninstall the pip version of mne to install dev version below
-    - |
-      git clone --depth 1 https://github.com/mne-tools/mne-python.git -b $MNE_VERSION
-      cd mne-python
-      pip install --no-deps -e .
-      cd ../
+    - if [ "${DEPS}" != "minimal" ]; then
+          conda env create -f $CONDA_ENV --name=testenv
+          source activate testenv
+          pip uninstall --yes --quiet mne  # uninstall the pip version of mne to install dev version below
+          git clone --depth 1 https://github.com/mne-tools/mne-python.git -b $MNE_VERSION
+          cd mne-python
+          pip install --no-deps -e .
+          cd ../
```

Comment:
I would pull these lines out of the if/else logic and put it below. Even for minimal dependencies, we might want to set the MNE version to be used with our `$MNE_VERSION` env variable